test/dynamo/test_subclasses.py

```diff
@@ -37,6 +38,32 @@ def setUpClass(cls):
     def tearDownClass(cls):
         cls._exit_stack.close()
 
+    def test_torch_function_state_tracing(self):
```

Comment:
Can you add a test where we disable it, graph break in the contents, and then have the correct state?